pyproject.toml

```
@@ -24,7 +24,7 @@ dependencies = [
   "numcodecs",
   "numpy",
   "ujson",
-  "zarr<3",
+  "zarr",
```

```suggestion
  "zarr>=3",
```
I think you'll  need to pin this to only >3 because a lot of the new syntax won't work with the 2.x API

---

Yes, agreed.